src/main/java/bio/terra/workspace/service/workspace/CloudSyncRoleMapping.java

```diff
@@ -0,0 +1,24 @@
+package bio.terra.workspace.service.workspace;
+
+import bio.terra.workspace.service.iam.model.IamRole;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import java.util.List;
+
+public class CloudSyncRoleMapping {
+  public static final ImmutableMap<IamRole, List<String>> cloudSyncRoleMap =
+      ImmutableMap.of(
+          IamRole.OWNER,
+              ImmutableList.of(
+                  "roles/viewer",
```

Comment:
I thought in your spec you had identified only roles/viewer on the project as the permission for all workspace roles. I didn't think we wanted uncontrolled objects created in the workspace project. For example, giving dataEditor on the project lets the workspace owner create BigQuery datasets in the project. Similarly, storage.objectAdmin.

Reply:
After more discussion with Greg we decided that we'll likely be letting users create their own resources in the workspace project for the initial pilot/CLI usage, especially before controlled resources are figured out. Agreed that it's incompatible with enforcing policies, and going to change in the future.